ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -220,13 +229,56 @@ export default class ExperimentalTab extends PureComponent {
     );
   }
 
+  renderTheme() {
+    if (!DARK_MODE_V1) {
+      return null;
+    }
+    const { t } = this.context;
+    const { theme, setTheme } = this.props;
+
+    const themesOptions = [
+      {
+        name: t('defaultTheme'),
+        value: THEME_TYPE.DEFAULT,
+      },
+      {
+        name: t('darkTheme'),
+        value: THEME_TYPE.DARK,
+      },
+    ];
+
+    return (
+      <div className="settings-page__content-row">
+        <div className="settings-page__content-item">
+          <span>
+            <I18nValue messageKey="theme" />
```

Comment:
[REQ] please do not use this component #13235

Reply:
Updated!  Good spotting!